.github/workflows/build-daily-no-build-cache.yml

```diff
@@ -10,19 +10,16 @@ jobs:
   common:
     uses: ./.github/workflows/build-common.yml
     with:
-      max-test-retries: 0
```

Comment:
I would think if we wanted to enable retries we would need to increase this value instead of remove it?

Reply:
See https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/edbd9aa8ab0ba9588a5fe21eff6ddc821cc0874e/conventions/src/main/kotlin/otel.java-conventions.gradle.kts#L369-L370

Reply:
thank you for pointing that out